spec/support/flipper.rb

```diff
@@ -0,0 +1,13 @@
+module Flipper
+  RSpec.configure do |config|
+    config.before(:each) do |example|
+      allow(Panoptes).to receive(:flipper).and_return(Flipper.new(Flipper::Adapters::Memory.new))
+      Panoptes.flipper["cellect"].enable
+      Panoptes.flipper["designator"].enable
+      Panoptes.flipper[:remove_complete_subjects].enable
```

Comment:
That's what Flipper is doing for us already. I imagined this as setting the default state we expect from Flipper, rather than all being disabled by default and enabled with a before(:each) at the top of every file.